src/ORM/FieldType/DBField.php

```diff
@@ -516,14 +516,6 @@ public function CDATA()
         return $this->XML();
     }
 
-    /**
-     * @return string
-     */
-    public function Nice()
```

Comment:
This is a semver breakage as Nice() isn't defined in parent class ViewableData.  At a practical level though this shouldn't matter.

I think this PR should target 5.2 instead since it's fixing a bug introduced in 5.2

Other option is to change this method to return `$this->->forTemplate()` to match what happens in [GridFieldDataColumns](https://github.com/silverstripe/silverstripe-framework/blob/5/src/Forms/GridField/GridFieldDataColumns.php#L268), though that seems messy.

Reply:
Never mind ignore me, not in 5.2 branch